src/main/java/baubles/common/items/ItemRing.java

```diff
@@ -18,33 +14,48 @@
 import net.minecraft.util.EnumHand;
 import net.minecraft.util.NonNullList;
 import net.minecraft.world.World;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
+import baubles.api.BaubleType;
+import baubles.api.BaublesApi;
+import baubles.api.IBauble;
+import baubles.api.cap.IBaublesItemHandler;
 
-public class ItemRing  extends Item implements IBauble
+@Mod.EventBusSubscriber
+public class ItemRing extends Item implements IBauble
 {
+	public static final Item RING = (new ItemRing()).setUnlocalizedName("Ring").setRegistryName("ring");
 
 	public ItemRing()
 	{
 		super();
 		this.setMaxStackSize(1);
 		this.setHasSubtypes(true);
 		this.setMaxDamage(0);
-		setCreativeTab(CreativeTabs.TOOLS);
+		this.setCreativeTab(CreativeTabs.TOOLS);
+	}
+
+	@SubscribeEvent
+	public static void registerItems(RegistryEvent.Register<Item> event) {
+		event.getRegistry().register(RING);
 	}
 
-	//TODO fix texture
 	@SideOnly(Side.CLIENT)
 	@Override
-	public void getSubItems(Item par1, CreativeTabs par2CreativeTab, NonNullList<ItemStack> par3NonNullList) {
-		par3NonNullList.add(new ItemStack(this,1,0));
+	public void getSubItems(CreativeTabs tab, NonNullList<ItemStack> list) {
```

Comment:
I went ahead and removed all `@SideOnly(Side.CLIENT)` annotations, as they seemed to all be unnecessary. And as was just discussed on `#minecraftforge`, modders should not be using it, unless absolutely necessary. I will push that tomorrow, after I have the chance to actually test that this doesn't break on a server after all though...